src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  setHideErrorBtn: (hide: boolean) => void;
+  hideError: boolean
```

Comment:
Consider naming boolean variables according to these rules
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/96409563/25ab9474-70a1-4bf9-8409-ec9cb4d0e09a)